editor.planx.uk/.storybook/preview.tsx

```diff
@@ -4,9 +4,14 @@ import { DndProvider } from "react-dnd";
 import { HTML5Backend } from "react-dnd-html5-backend";
 import { defaultTheme } from "../src/theme";
 import React from "react";
+import { MyMap } from "@opensystemslab/map";
 
 import { reactNaviDecorator } from "./__mocks__/react-navi";
 
+if (!window.customElements.get("my-map")) {
+  window.customElements.define("my-map", MyMap);
+}
```

Comment:
Required to view the web component in Storybook.

Reply:
:bulb: this is great! I actually thought this was a limitation of _React_ Storybook and not possible to render native web components - excited to backfill and correctly configure other geocomponents too :raised_hands: